dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -0,0 +1,275 @@
+		...format,
+		theme: getCurrentPillar(CAPIArticle),
+	};
+
+	const {
+		config: { host },
+	} = CAPIArticle;
+
+	const adTargeting: AdTargeting = buildAdTargeting({
+		isAdFreeUser: CAPIArticle.isAdFreeUser,
+		isSensitive: CAPIArticle.config.isSensitive,
+		videoDuration: CAPIArticle.config.videoDuration,
+		edition: CAPIArticle.config.edition,
+		section: CAPIArticle.config.section,
+		sharedAdTargeting: CAPIArticle.config.sharedAdTargeting,
+		adUnit: CAPIArticle.config.adUnit,
+	});
+	const contributionsServiceUrl = getContributionsServiceUrl(CAPIArticle);
+
+	return (
+		<>
+			<div data-print-layout="hide" id="bannerandheader">
+				<>
+					<Stuck>
+						<ElementContainer
+							showTopBorder={false}
+							showSideBorders={false}
+							padded={false}
+							shouldCenter={false}
+						>
+							<HeaderAdSlot
+								isAdFreeUser={CAPIArticle.isAdFreeUser}
+								shouldHideAds={CAPIArticle.shouldHideAds}
+								display={format.display}
+							/>
+						</ElementContainer>
+					</Stuck>
+					<ElementContainer
+						showTopBorder={false}
+						showSideBorders={false}
+						padded={false}
+						backgroundColour={brandBackground.primary}
+						element="header"
+					>
+						<Header
+							edition={CAPIArticle.editionId}
+							idUrl={CAPIArticle.config.idUrl}
+							mmaUrl={CAPIArticle.config.mmaUrl}
+							discussionApiUrl={
+								CAPIArticle.config.discussionApiUrl
+							}
+							isAnniversary={
+								CAPIArticle.config.switches.anniversaryHeaderSvg
+							}
+							urls={CAPIArticle.nav.readerRevenueLinks.header}
+							remoteHeader={
+								CAPIArticle.config.switches.remoteHeader
+							}
+							contributionsServiceUrl={contributionsServiceUrl}
+						/>
+					</ElementContainer>
+					<ElementContainer
+						showSideBorders={true}
+						borderColour={brandLine.primary}
+						showTopBorder={false}
+						padded={false}
+						backgroundColour={brandBackground.primary}
+						element="nav"
+					>
+						<Nav
+							nav={NAV}
+							format={formatForNav}
+							subscribeUrl={
+								CAPIArticle.nav.readerRevenueLinks.header
+									.subscribe
+							}
+							edition={CAPIArticle.editionId}
+						/>
+					</ElementContainer>
+
+					<ElementContainer
+						backgroundColour={palette.background.article}
+						padded={false}
+						element="aside"
+					>
+						<Island deferUntil="idle">
+							<SubNav
+								subNavSections={newslettersSubNav}
+								currentNavLink={NAV.currentNavLink}
+								format={format}
+							/>
+						</Island>
+					</ElementContainer>
+					<ElementContainer
+						backgroundColour={palette.background.article}
+						padded={false}
+						showTopBorder={false}
+					>
+						<Lines count={4} effect="straight" />
+					</ElementContainer>
+				</>
+			</div>
+
+			{CAPIArticle.config.switches.surveys && (
+				<AdSlot position="survey" display={format.display} />
+			)}
+
+			<main>
+				<ElementContainer padded={false}>
+					<NewsLetterSignupBanner />
```

Comment:
The logo in the  banner is part of the layout design rather than the `MainMedia` for the article (on the design, the madin media will be placed alongside the content - like the "first Edition"graphic on https://www.theguardian.com/global/ng-interactive/2022/apr/13/first-edition-sign-up-guardian.

I'll see if I can get a smaller version of the file - 32kb is more than it should be.

Reply:
Ah, I see now. Thanks for explaining. Sideways main media, cool!

Re the svg. I notice that the Guardian logo is in there. We already have that SVG in the source system so we could just import that from there - it's already being imported in the Header. I also see the yellow Newsletters text gets positioned in different places based on breakpoint so having them separate makes sense. I wonder in that 'Newsletters' could just be...text? It's using the font we already have on the page so unless there's a requirement for it to be an svg I think text would be much smaller and easier to work with here?